examples/distributed-loki/alerting/loki-ratelimit-alert.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: loki-alerts
+  namespace: openshift-operators-redhat
+spec:
+  groups:
+  - name: LokiRateLimitAlerts
+    rules:
+    - alert: LokiTenantRateLimit
+      annotations:
+        message: |-
+          {{ $labels.job }} {{ $labels.route }} is experiencing 429 errors.
+        summary: "At any number of requests are responded with the rate limit error code."
+      expr: sum(irate(loki_request_duration_seconds_count{status_code="429"}[1m])) by (job, namespace, route) / sum(irate(loki_request_duration_seconds_count[1m])) by (job, namespace, route) * 100 > 0
```

Comment:
I think the metric should be `netobserv_loki_request_duration_seconds_count` (missing the netobserv prefix), right?

Reply:
I didn't realize there's `netobserv` (client side) metric as well, I had set it up for Loki metric in `openshift-operators-redhat` NS. Do you think this Rule would be appropriate for client side metric `netobserv_loki_request_duration_seconds_count` instead?

Reply:
ah ok I didn't see the other one.
Yeah I would rather use the netobserv / client one for two reasons:
- it's only for netobserv (the other one could be used as well if you deploy a loki stack for logging .. though it's surely possible to discriminate by namespace)
- it also work when not using the loki operator